src/main/scala/com/gu/support/workers/model/User.scala

```diff
@@ -8,9 +8,9 @@ case class User(
   firstName: String,
   lastName: String,
   country: Country,
-  state: Option[String],
-  allowMembershipMail: Boolean,
-  allowThirdPartyMail: Boolean,
-  allowGURelatedMail: Boolean,
-  isTestUser: Boolean
+  state: Option[String] = None,
+  allowMembershipMail: Boolean = false,
+  allowThirdPartyMail: Boolean = false,
+  allowGURelatedMail: Boolean = false,
```

Comment:
Are these used in general (e.g. in Identity), but just not relevant for support-workers?

Reply:
They are completely unused since GDPR, we now deal with Identity directly from support-frontend